.github/ISSUE_TEMPLATE/SUPPORT.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Support, Help, and Advice
+about: Need help or support? Please don't open an issue! Head to https://stackoverflow.com/questions/tagged/chart.js.
+labels: 'type: support'
+---
+
+Hey there! If you need help or tech support then this is not the place to
+ask. Please post a question to [StackOverflow](https://stackoverflow.com/questions/tagged/chart.js)
+or head over to [Chart.js Slack](https://chartjs-slack.herokuapp.com/).
+
+You maximize your chances of having your question answered if you follow
+[StackOverflow's guidelines for how to ask a good question](https://stackoverflow.com/help/how-to-ask).
+If your question was not answered on [StackOverflow](https://stackoverflow.com/questions/tagged/chart.js),
+that does not mean you should post it here. If you do, it will be immediately closed.
+
+Chart.js users vastly outnumber our small number of volunteer maintainers and we try to help as many users
+as we can, but we may not get to every question. Chart.js is a community project. You can contribute to maximizing
+the number of users helped by
+[asking good questions](https://stackoverflow.com/help/how-to-ask),
+answering questions on [StackOverflow](https://stackoverflow.com/questions/tagged/chart.js)
+or [Slack](https://chartjs-slack.herokuapp.com/),
+creating an issue or pull request for the documentation,
+or otherwise [contributing to the project](https://www.chartjs.org/docs/latest/developers/contributing.html).
```

Comment:
I like the formatting of `ISSUE_TEMPLATE.md`, should we use the same here (between `<!-- -->` with line breaks around 80 characters)? I would also remove markdown formatted links (and duplicated links) because they don't appear when creating tickets.

**Suggestion:**

```
<!--
  Hey there! 

  If you need help or tech support then this is not the place to ask Please post
  a question to https://stackoverflow.com/questions/tagged/chart.js or head over
  to our Slack workspace (https://chartjs-slack.herokuapp.com/).

  You maximize your chances of having your question answered if you follow
  StackOverflow's guidelines: https://stackoverflow.com/help/how-to-ask.
  If your question was not answered on StackOverflow, that does not mean you 
  should post it here. If you do, it will be immediately closed.

  Chart.js users vastly outnumber our small number of volunteer maintainers and 
  we try to help as many users as we can, but we may not get to every question. 
  Chart.js is a community project. You can contribute to maximizing the number of
  users helped by asking good questions and answering questions on StackOverflow
  or Slack, creating an issue or pull request for the documentation, or otherwise
  contributing to the project:

  https://www.chartjs.org/docs/latest/developers/contributing.html
-->
```

![image](https://user-images.githubusercontent.com/3874900/51020957-640bc180-1580-11e9-87f8-85e7ddfcb546.png)

**Currently:**

![image](https://user-images.githubusercontent.com/3874900/51021001-8998cb00-1580-11e9-8dd7-9e8e68abb6fe.png)

Reply:
Done. Looks much better. Thanks